src/components/FAQ.jsx

```
@@ -1,173 +1,168 @@
-              FREQUENTLY ASKED QUESTIONS
-            </h1>
-            <p className="text-gray-300 text-lg max-w-2xl mx-auto">
-              Everything you need to know about joining and being part of the APP Team
-            </p>
+      <div className="max-w-6xl mx-auto">
+        
+        {/* --- Header --- */}
+        <div className="mb-20 flex flex-col md:flex-row justify-between items-end border-b-2 border-[#f4f4f5] pb-8">
+          <div>
+             <span className="text-[#ccff00] font-bold tracking-widest text-xs uppercase mb-2 block">
+                // System Queries
```

Biome (2.1.2)

[error] Wrap comments inside children within braces.

Unsafe fix: Wrap the comments with braces


(lint/suspicious/noCommentText)